build.gradle.kts

```diff
@@ -239,22 +227,23 @@ subprojects {
 
     tasks {
         registerTestTasks()
-        test {
+        test.configure {
```

Comment:
After discussion, we decided to leave this cumbersome `.configure { ... }` since it's a part of [Gradle's recommendations](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html).